src/dbseeder/dbseeder.py

```diff
@@ -87,28 +92,47 @@ def post_process(self, who):
         print('joining to layer')
         arcpy.AddJoin_management(stationsLyr, 'Id', stationsIdent, 'FID_' + stations_fc)
 
-        print('calculating fields')
+        print('calculating state')
         arcpy.CalculateField_management(stationsLyr, 'StateCode', '!STATE_FIPS!', 'PYTHON')
+        print('calculating county')
         arcpy.CalculateField_management(stationsLyr, 'CountyCode', '!FIPS!', 'PYTHON')
 
         print('removing join')
         arcpy.RemoveJoin_management(stationsLyr, stations_identity)
 
         #: Elevation
-        print('selecting points with null values')
-        arcpy.SelectLayerByAttribute_management(stationsLyr, where_clause='Elev IS NULL')
-
-        print('extracting values to points')
-        arcpy.CheckOutExtension('Spatial')
-        dem_points = arcpy.sa.ExtractValuesToPoints(stationsLyr, dem, r'in_memory\DEMPoints', add_attributes='VALUE_ONLY')
-
-        print('joining to layer')
-        arcpy.AddJoin_management(stationsLyr, 'StationId', dem_points, 'StationId')
-
-        print('calulcating fields')
-        arcpy.CalculateField_management(stationsLyr, 'Elev', '!RASTERVALU!', 'PYTHON')
-        arcpy.CalculateField_management(stationsLyr, 'ElevUnit', '"meters"')
-        arcpy.CalculateField_management(stationsLyr, 'ElevMeth', '"Other"')
+        print('looping through points with null elevation values')
+        connection = pyodbc.connect(self._get_db(who)['connection_string'])
+        cursor = connection.cursor()
+        cursor.execute('''
+            SELECT Lon_X, Lat_Y, Id
+            FROM Stations
+            WHERE Elev IS NULL OR Elev = 0 OR Elev > 20000
+        ''')
+        i = 0
+        batch_size = 100
+        rows = cursor.fetchall()
+        total = len(rows)
+        for row in rows:
+            payload = {'x': row.Lon_X, 'y': row.Lat_Y, 'units': 'Meters', 'output': 'json'}
+            r = requests.get(epqs_service_url, params=payload)
+            try:
+                elev = r.json()['USGS_Elevation_Point_Query_Service']['Elevation_Query']['Elevation']
+            except:
+                print('error retrieving elevation for Lon: {} & Lat: {}. Skipping'.format(row.Lon_X, row.Lat_Y))
+                continue
+
+            unit = 'meters'
+            method = 'Other'
+            cursor.execute('''
```

Comment:
this should use the [pyodbc params](https://code.google.com/p/pyodbc/wiki/Cursor#execute) instead of string formatting.